grafana-plugin/src/pages/integration/Integration.tsx

```diff
@@ -827,8 +828,9 @@ const IntegrationActions: React.FC<IntegrationActionsProps> = ({
     onConfirm: () => void;
   }>(undefined);
 
+  const [isServiceNowConfigOpen, setServiceNowConfigOpen] = useState(false);
```

Comment:
Take a look on `useDrawer` hook I created.  It would be great to start using it so that handling drawers is consistent

Reply:
Done